app/views/vendor_outage/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render(
+      'idv/shared/error',
+      title: t('vendor_outage.working'),
+      heading: t('vendor_outage.working'),
+      options: [
+        {
+          text: t('vendor_outage.get_updates_on_status_page'),
+          url: StatusPage.base_url,
+          new_tab: true,
+        },
+        {
+          text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
+          url: MarketingSite.contact_url,
+          new_tab: true,
+        },
+      ],
+    ) do %>
+  <p><%= @specific_message %></p>
+<% end %>
+
+<%= render(
+      'idv/doc_auth/back',
+      fallback_path: root_path,
```

Comment:
For all the idv cases, this seems to be the correct destination, but that's likely not going to always be the case.

Reply:
> For all the idv cases, this seems to be the correct destination, but that's likely not going to always be the case.

Hm, since the back partial will just not render a "Back" button if it can't determine a path, maybe that would be preferable than having it show and linking to the root path?

i.e.

```suggestion
```

Reply:
My problem was that the default determination of a path was to go back to `/verify/doc_auth/welcome` which got redirected back to the outage page. But now that I've tried this out again... it works great! 😛 

I think I had `step: ...` set previously maybe? Doesn't matter. Sounds good!